package.json

```diff
@@ -95,34 +88,35 @@
   },
   "devDependencies": {
     "@pdf-lib/fontkit": "^1.1.0",
-    "@rollup/plugin-commonjs": "^13.0.0",
-    "@rollup/plugin-json": "^4.1.0",
-    "@rollup/plugin-node-resolve": "^8.0.1",
-    "@types/jest": "^26.0.0",
-    "@types/node-fetch": "^2.5.7",
+    "@rollup/plugin-commonjs": "^23.0.2",
+    "@rollup/plugin-json": "^6.1.0",
+    "@rollup/plugin-node-resolve": "^14.1.0",
+    "@types/node-fetch": "^2.6.13",
     "@types/pako": "^1.0.1",
     "@zerollup/ts-transform-paths": "^1.7.18",
-    "downlevel-dts": "^0.5.0",
+    "downlevel-dts": "^0.11.0",
     "flamebearer": "^1.1.3",
-    "http-server": "^0.12.3",
-    "jest": "^26.0.1",
-    "node-fetch": "^2.6.0",
-    "prettier": "^2.0.5",
+    "node-fetch": "^2.6.7",
+    "prettier": "^2.8.8",
     "rimraf": "^3.0.2",
-    "rollup": "^2.17.1",
+    "rollup": "2.79.2",
     "rollup-plugin-terser": "^6.1.0",
-    "ts-jest": "^26.1.0",
-    "tslint": "^6.1.2",
-    "tslint-config-prettier": "^1.18.0",
-    "ttypescript": "^1.5.10",
+    "ttypescript": "^1.5.15",
     "typescript": "^3.9.5"
   },
+  "resolutions": {
+    "minimist": "^1.2.6",
+    "shelljs": "^0.8.5"
```

Comment:
is this required? just checking

Reply:
using https://github.com/ParadoxAi/pdf-lib fork's package.json to get latest deps